packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -96,4 +96,9 @@ class BottomNavigationBarItem {
   ///  * [Icon.color] and [ImageIcon.color] to control the foreground color of
   ///    the icons themselves.
   final Color? backgroundColor;
+
+  /// The text to display in the tooltip for this [BottomNavigationBarItem].
+  ///
+  /// The tooltip will only appear on an item in the [BottomNavigationBar].
```

Comment:
maybe:

The text to display in the [Tooltip] for this [BottomNavigationBarItem], when the user long presses the item.

The [Tooltip] will only appear on an item in a Material design [BottomNavigationBar], and when the string is not empty. 

Defaults to null, in which case the [label] text will be used.

Reply:
What should the bottom navigation bar do if both `toolTip` and `label` are null?

Reply:
The use cases are : 
* If icon and title are provided - No error and no tooltip will be shown.
* If only icon is provided - Error will be thrown https://github.com/flutter/flutter/blob/d075b979f86dd41d49f250958ba486f30512d43a/packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart#L36
* if icon and label are provided - No error and tooltip with `label` will be shown.
* if icon, label and tooltip are provided - No error and tooltip with `tooltip` will be shown.